ci/Dockerfile

```diff
@@ -35,16 +35,14 @@ RUN apk add --no-cache --no-progress --update --upgrade \
         && \
     mv /etc/apk/repositories /etc/apk/repositories.old && \
     apk add --no-cache --update --upgrade \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/main \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.11/community \
+        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.12/main \
+        --repository=http://dl-cdn.alpinelinux.org/alpine/v3.12/community \
         lshw \
         && \
-    mv /etc/apk/repositories.old /etc/apk/repositories && \
-    apk add --no-cache --update --upgrade \
-        --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing \
-        cfssl \
+    mv /etc/apk/repositories.old /etc/apk/repositories \
```

Comment:
I'm not familiar with our style guidance here - but the line continuation here is unnecessary and the trailing semi-colon is highly irregular. Can both be removed?